README

```diff
@@ -1,5 +1,11 @@
-This is beanstalkd, a fast, general-purpose work queue.
-See http://kr.github.io/beanstalkd/ for general info.
+This is a fork of kr/beanstalkd, a fast, general-purpose work queue focusing
+on features and fixes addressing performance and production critical problems.
+
+
+ADDED FEATURES AND FIXES
```

Comment:
Do you mind keeping these changes out of the PR; I can't merge them. Of course, it still makes sense to update the readme on your fork, but I guess it should be on another branch.